.editorconfig

```diff
@@ -4,6 +4,6 @@ root = true
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-[*.{c,cpp,h,hpp}]
+[*.{c,cpp,h,hpp,mm}]
```

Comment:
I'd really like this to be a separate commit, tbh. Fixing the editor config is logically separate from the UI logic change.

Reply:
Separate commit in this PR, or also separate PR?

Reply:
Same PR is fine, unless you want to aim for claiming #10000 😄

Reply:
I’d need to sit out two more issues/PRs 😇